src/Microsoft.OpenApi.OData.Reader/Operation/EntityGetOperationHandler.cs

```diff
@@ -30,14 +30,17 @@ protected override void SetBasicInfo(OpenApiOperation operation)
             // Summary
             operation.Summary = "Get entity from " + EntitySet.Name + " by key";
 
+            IEdmEntityType entityType = EntitySet.EntityType();
+
```

Comment:
There's a line to set the description in the base class.

I don't know why do you want to rewrite it using the entity type. 

Maybe you need to change that logic:

1) If we have description for the entity set, use it
2) otherwise, if we have description for entity type, use it
3) otherwise, leave it empty or use a default string.

Reply:
Are we not expecting a scenario where we have descriptions for both entity set and entity? In such a scenario, we would end up setting entity set descriptions for entities, even though they have their own annotated descriptions.

Reply:
You mean for "single entity request, the annotation should from entity type?".

Reply:
Yes.

Reply:
Typically, it should comes from entity set, not from entity type.
However, it seems we can't distinguish it from entity set or single entity.
So, I am ok for your changes.